src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts

```diff
@@ -81,8 +81,9 @@ export class GroupProgressGridComponent implements OnChanges, OnDestroy {
   dialog: 'loading'|'opened'|'closed' = 'closed';
   dialogTitle = '';
 
-  private dataFetching = new Subject<{ groupId: string, itemId: string, attemptId: string, filter: TypeFilter }>();
+  private dataFetching = new ReplaySubject<{ groupId: string, itemId: string, attemptId: string, filter: TypeFilter }>(1);
```

Comment:
ok, I have to check this.

Reply:
That's good to have a replay subject here for the initial fetching info. So that's ok.